package-system/AWSGameLiftServerSDK/README.md

```diff
@@ -3,115 +3,30 @@
-   ```
-   $ sudo ln -s /usr/bin/clang-6.0 /usr/bin/clang
-   ```
-   ```
-   $ sudo ln -s /usr/bin/clang++-6.0 /usr/bin/clang++ 
-   ```
-
-4. Patch `External_boost.cmake` under `GameLift-Cpp-ServerSDK-{version-number}/cmake`
-   to make sure boost library get build with correct toolset (clang-6.0 for example) and variants
-   * Under `GameLift-Cpp-ServerSDK-{version-number}` directory, run command `git init` to init a local git repository with current directory
-   * Run command `git add .` to stage all existing files
-   * Run command `git commit -m "init commit"` to commit your existing files
-   * Run command `git apply --ignore-space-change --ignore-whitespace AWSGameLiftServerSDK_3.4.1.patch` to apply patch 
-   * Run command `git diff` to check expected changes like
-     ```diff
-          BUILD_COMMAND b2 address-model=${am} ${boost_with_args}
-        )
-      else()
-     +  if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-     +    set(_build_variant "debug")
-     +  else()
-     +    set(_build_variant  "release")
-     +  endif()
-     +
-        list(APPEND boost_with_args
-     -    "cxxflags=-fPIC")
-     +    "cxxflags=-fPIC" "toolset=clang-6.0" "variant=${_build_variant}")
-        set(boost_cmds
-          CONFIGURE_COMMAND ./bootstrap.sh --prefix=<INSTALL_DIR>
-          BUILD_COMMAND ./b2 address-model=${am} ${boost_with_args}
-     ```
-
-5. Run the python script `build_and_package.py` under `GameLift-Cpp-ServerSDK-{version-number}` directory, like
-   ```
-   & python3 build_and_package.py --config gamelift-sdk.json --platform linux --version 3.4.1
-   ```
-
-6. Build package will be located under `GameLift-Cpp-ServerSDK-{version-number}/build/{version-number}` directory
-
-7. Copy and put build package into `package-system/AWSGameLiftServerSDK/linux/AWSGameLiftServerSDK`, for example
-   ``` 
-   - AWSGameLiftServerSDK
-     - include
-     - lib
-     - LICENSE_AMAZON_GAMELIFT_SDK.txt
-     - NOTICE_C++_AMAZON_GAMELIFT_SDK.txt
-   ```
-8. Update `package-system/AWSGameLiftServerSDK/linux/PackageInfo.json` with built version `PackageName`
-
-9. Update `<3p-package-source path>/package_build_list_host_linux.json` with updated `PackageName` in step 6
+Link Clang compiler for cmake build, for example:
+```
+$ sudo ln -s /usr/lib/llvm-12/bin/clang /usr/bin/clang
+```
+```
+$ sudo ln -s /usr/lib/llvm-12/bin/clang++ /usr/bin/clang++ 
+```
 
 ## FAQ
-* For windows build, use `--msbuild_path` argument to specify msbuild executable path if msbuild is unrecognized as a valid command, like
-  ```
-  & python build_and_package.py --config gamelift-sdk.json --platform windows --version 3.4.1 --msbuild_path "C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\15.0\Bin\MSBuild.exe"
-  ```
-
+* For windows build, as dependent boost version is old and not well compatible with Visual Studio 2019, feel free to switch to `Visual Studio 15 2017` if necessary 
```

Comment:
Avoid ambiguous instructions. Can then build with VS2019 or is the recommendation to use VS2017? Maybe "Because the Amazon GameLift Server SDK depends on an older version of boost, it is recommend that you build to VS2017" or similar. Unclear what problems they may encounter or when then should switch.

Reply:
will update